styleguide.config.js

```diff
@@ -14,8 +15,8 @@ module.exports = {
   },
   skipComponentsWithoutExample: true,
   styleguideComponents: {
-    Wrapper: path.join(__dirname, 'styleguide/ThemeWrapper'),
+    Wrapper: path.join(__dirname, 'styleguide/Wrapper'),
   },
   usageMode: 'expand',
-  webpackConfig: createConfig([babel()]),
+  webpackConfig: createConfig([babel(), setEnv(dotenv.config().parsed)]),
```

Comment:
What problem is that solving?

Reply:
We want to be able to show the stripe input in styleguide, thus we need the `STRIPE_KEY` env variable. Loading the `.env` config in styleguide allows us to do this.

Reply:
So we don't load the `.env` anymore, instead we now read the env but fallback on a default value - the same way that we do on the frontend.